scripts/travis/install-caffe.sh

```diff
@@ -11,9 +11,9 @@ then
 fi
 
 INSTALL_DIR=$1
-NUM_THREADS=${NUM_THREADS-4}
-CAFFE_URL=https://github.com/NVIDIA/caffe.git
-CAFFE_BRANCH=caffe-0.14
+NUM_THREADS=${NUM_THREADS:-4}
```

Comment:
Seems to work. I got it from here:
http://unix.stackexchange.com/questions/122845/using-a-b-for-variable-assignment-in-scripts